Colony_mask_training_inference/run.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+set -e  # Exit immediately if a command exits with a non-zero status
+set -o pipefail  # Makes pipelines return the exit code of the last failing command
+
+DEVICE_ID="add/your/GPU/UUID"
+CONFIG_PATH="$PWD/configs"
+EVAL_SCRIPT="../../cyto-dl/cyto_dl/eval.py"
+
+# List of experiments to run sequentially
+EXPERIMENTS=( 
+    #"im2im/yamls_generated/eval_scale2_7450_p10.yaml"
+    #"im2im/yamls_generated/eval_scale2_7450_p11.yaml"
+    #"im2im/yamls_generated/eval_scale2_7450_p12.yaml"
+    #"im2im/yamls_generated/eval_scale2_7450_p13.yaml"
+)
+
+echo "Using device: $DEVICE_ID"
+echo "Using config path: $CONFIG_PATH"
+
+for EXP in "${EXPERIMENTS[@]}"; do
```

Comment:
```suggestion
CONFIG_PATH="$PWD/configs"
EVAL_SCRIPT="../../cyto-dl/cyto_dl/eval.py"

echo "Using device: $DEVICE_ID"
echo "Using config path: $CONFIG_PATH"

for EXP in EXPERIMENTS_DIR/*.yaml; do
```

Ideally, the user should not be expected to edit any code. In this case, the user inputs DEVICE_ID and EXPERIMENTS can just be environment variables instead of written in `run.sh`. With this suggestion, the appropriate way to run the script would be `DEVICE_ID=<your device ID> EXPERIMENTS_DIR=<path to your folder of generated YAMLS run.sh`.

Since the EXPERIMENTS_DIR is the output of a previous step, perhaps it can be assumed to be `im2im/yamls_generated` and no user input is needed for that one? In that case the following suggestion would be preferable:
```suggestion
CONFIG_PATH="$PWD/configs"
EVAL_SCRIPT="../../cyto-dl/cyto_dl/eval.py"
EXPERIMENTS_DIR="im2im/yamls_generated"

echo "Using device: $DEVICE_ID"
echo "Using config path: $CONFIG_PATH"

for EXP in EXPERIMENTS_DIR/*.yaml; do
```

In either case, the README should be updated to reflect these changes.